app/renderer/js/pages/preference/new-server-form.ts

```diff
@@ -30,15 +30,21 @@ class NewServerForm extends BaseComponent {
 					</div>
 				</div>
 				<div class="server-center">
-				<div class="divider">
-					<hr class="left"/>${t.__('OR')}<hr class="right" />
+					<div class="divider">
+						<hr class="left"/>${t.__('OR')}<hr class="right" />
+					</div>
 				</div>
+				<div class="server-center">
+					<div class="server-save-action">
+						<button id="open-create-org-link">${t.__('Create a new organization')}</button>
+					</div>
 				</div>
 				<div class="server-center">
-				<div class="server-save-action">
-					<button id="open-create-org-link">${t.__('Create a new organization')}</button>
-			</div>
+					<div class="server-network-option">
+						<span id="open-network-settings"> Network and Proxy Settings</span>
```

Comment:
You need to add the translate function here. `$t.('')}`

Reply:
Done !